app/views/contribute/index.erb

```diff
@@ -135,11 +135,11 @@
           <h2>Make Up Exercises</h2>
 
           <p>
-            We're always looking to add new exercises. Have an idea? Suggest it in the <a href="https://github.com/exercism/x-common">metadata repository</a>.
+            We're always looking to add new exercises. Have an idea? Suggest it in the <a href="https://github.com/exercism/problem-specifications">metadata repository</a>.
           </p>
 
            <p>
-            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">implementing a new exercise</a> in the Contributing Guide.
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">implementing a new exercise</a> in the Contributing Guide.
```

Comment:
This lives here: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/implementing-a-completely-new-exercise.md